runtime/amd64.S

```diff
@@ -588,9 +588,9 @@ ENDFUNCTION(G(caml_call_realloc_stack))
 
 FUNCTION(G(caml_call_gc))
 CFI_STARTPROC
+LBL(caml_call_gc):
```

Comment:
Nooooooooo! The label should not be moved over the `ENTER_FUNCTION` macro, this will break things. But the `CFI_SIGNAL_FRAME` can probably get moved after `ENTER_FUNCTION` (and after the label).

Reply:
Another possibility would be to keep `CFI_SIGNAL_FRAME` above `ENTER_FUNCTION` and add `CFI_SIGNAL_FRAME` to the code paths which jump to `LBL(caml_call_gc)`.

Reply:
I'm retracting my changes to `amd64.S` and just focusing on fixing the other backends mentioned.